node/src/chain_spec/turing.rs

```diff
@@ -106,35 +111,49 @@ pub fn turing_latest_latest() -> ChainSpec {
 		"turing",
 		ChainType::Live,
 		move || {
-			let allocation_json = &include_bytes!("../../../distribution/neumann_alloc.json")[..];
+			let allocation_json = &include_bytes!("../../../distribution/turing_alloc.json")[..];
 			let initial_allocation: Vec<(AccountId, Balance)> =
 				serde_json::from_slice(allocation_json).unwrap();
+			const ALLOC_TOKENS_TOTAL: u128 = DOLLAR * 58_000_000;
+			validate_allocation(initial_allocation.clone(), ALLOC_TOKENS_TOTAL, EXISTENTIAL_DEPOSIT);
+
+			let vesting_json = &include_bytes!("../../../distribution/turing_vesting.json")[..];
+			let initial_vesting: Vec<(u64, Vec<(AccountId, Balance)>)> =
+				serde_json::from_slice(vesting_json).unwrap();
 
-			validate_allocation(initial_allocation.clone(), TOTAL_TOKENS, EXISTENTIAL_DEPOSIT);
+			let vested_tokens = 9_419_999_999_999_999_919;
+			let vest_starting_time: u64 = 1651777200;
+			let vest_ending_time: u64 = 1743879600;
+			validate_vesting(initial_vesting.clone(), vested_tokens, EXISTENTIAL_DEPOSIT, vest_starting_time, vest_ending_time);
+
+			let collator_bond = 400_000 * DOLLAR;
 
 			testnet_genesis(
 				// initial collators.
 				vec![
 					(
-						// 5ECasnYivb8cQ4wBrQsdjwRTW4dzJ1ZcFqJNCLJwcc2N6WGL
-						hex!["5e7aee4ee53ef08d5032ba5db9f7a6fdd9eef52423ac8c1aa960236377b46610"]
+						// SS58 prefix 51: 68eS5bmdz2nY45KxBfBFePVH88xupaPvjzExMxcnGuRBLcSK
+						// SS58 prefix substrate: 5CdDSsMpHjCkdqX49iyaozNrnLEZ4xUkk2mX4N6cVbxxpYhy
+						hex!["6e6dd706f9c325ea3d316392135766c0b16feb4fd6fa3171ae2da5f00f022a0a"]
 							.into(),
-						hex!["5e7aee4ee53ef08d5032ba5db9f7a6fdd9eef52423ac8c1aa960236377b46610"]
+						hex!["6e6dd706f9c325ea3d316392135766c0b16feb4fd6fa3171ae2da5f00f022a0a"]
 							.unchecked_into(),
 					),
 					(
-						// 5D2VxzUBZBkYtLxnpZ9uAV7Vht2Jz5MwqSco2GaqyLwGDZ4J
-						hex!["2a8db6ca2e0cb5679e0eff0609de708c9957f465af49abbe7ff0a3594d52933e"]
+						// SS58 prefix 51: 6BKgyjBS7exBgGKCVMGbYYcdTLHnXdJFHXD86Vr3vX4vpzPz
+						// SS58 prefix substrate: 5HWVKNhY1PcSWcpSX5QgXJ7iYxJkAubDxxBB4p2v4wBRXQkE
+						hex!["e4d7080396ff3c0d501d18d33afee4e3140a6a878e6277968bf3771fe6cd4956"]
 							.into(),
-						hex!["2a8db6ca2e0cb5679e0eff0609de708c9957f465af49abbe7ff0a3594d52933e"]
+						hex!["e4d7080396ff3c0d501d18d33afee4e3140a6a878e6277968bf3771fe6cd4956"]
 							.unchecked_into(),
 					),
 				],
 				// 5GcD1vPdWzBd3VPTPgVFWL9K7b27A2tPYcVTJoGwKcLjdG5w
 				hex!["c8f7b3791290f2d0f66a08b6ae1ebafe8d1efff56e31b0bb14e8d98157379028"].into(),
```

Comment:
Wrong sudo for turing `08df8338e854d8d589dedd4305c11e589cbef994e5dd00c7bb8fb7d277705b06`